src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,49 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String report = createReport(fromFileName);
+        writeTofile(report, toFileName);
+    }
+
+    private void writeTofile(String report, String toFileName) {
+        File file = new File(toFileName);
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(file))) {
+            bufferedWriter.write(report);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write data to file", e);
+        }
+    }
 
+    private String createReport(String fromFileName) {
+        File file = new File(fromFileName);
+        int supply = 0;
+        int buy = 0;
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(file));
```

Comment:
see checklist please - Remember, if you are using classes that implement an AutoCloseable interface, we should use it with try-with-resources.

Reply:
done